packages/cli/src/utils/index.ts

```diff
@@ -163,7 +163,7 @@ export function mergeSteps(prevSteps: Step[], currentSteps: Step[]) {
 
     // If previous step with the same commit exists, copy it.
     // Or just return the new step.
-    return prevStep || currentStep;
+    return prevStep ? { ...prevStep, ...currentStep } : currentStep;
```

Comment:
This will erase everything in the `prevStep`, including `articleId` and `explain`.